arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -1298,6 +1298,22 @@ Result RocksDBEngine::writeCreateCollectionMarker(TRI_voc_tick_t databaseId,
   return rocksutils::convertStatus(res);
 }
 
+Result RocksDBEngine::writeCreateCollectionMarkerBatch(TRI_voc_tick_t databaseId,
+                                                  DataSourceId cid, VPackSlice const& slice,
+                                                  RocksDBLogValue&& logValue, rocksdb::WriteBatch& batch) {
+  RocksDBKey key;
+  key.constructCollection(databaseId, cid);
+  auto value = RocksDBValue::Collection(slice);
+
+  if (logValue.slice().empty()) {
```

Comment:
Should this condition be negated here?
```suggestion
  if (!logValue.slice().empty()) {
```